### DISCUSSION TITLE
New Cell voltages page 0.12

### DISCUSSION THREAD

**Discussion by CRaccani** [Louisvdw/dbus-serialbattery#192] [Q&A] [answered]
Hi Lois, thank you for this great feature.
I have a simple query: Is the Cell voltages page only available to Daly BMS's?
I have a JKBMS, does not appear in latest version beta either, 0.12b2.

Thanks.

**Comment by Louisvdw** [comment, marked as answer]
Hi @CRaccani 
To see the cell voltages you need to run an extra step to see them in the display.
Log in to your GX using SSH and run

> sh /data/etc/dbus-serialbattery/installqml.sh

**Reply by CRaccani** [reply]
Thank you, that did it.
This is fantastic.
